tests/data/inverse-data.json

```diff
@@ -0,0 +1,5276 @@
+[
```

Comment:
Are you sure you need all these cases? It looks like a many of them are testing identical things? Let's try to keep only one tests of each kind.